packages/mdc-fab/_mixins.scss

```diff
@@ -305,6 +312,17 @@ $mdc-fab-icon-enter-duration_: 180ms;
   }
 }
 
+@mixin mdc-fab--touch_($query: mdc-feature-all()) {
+  @include mdc-touch-target-component(
```

Comment:
Not in the scope of this PR but would it make sense to rename `mdc-touch-target-component()` to `mdc-touch-target-margin()`? I think this mixin is only applying margin to increase its touch size.

WDYT?

Reply:
Makes sense! I'll do this in a follow-up CL/PR.